src/app/components/start-load-test-widget/start-load-test-widget.component.ts

```diff
@@ -47,10 +52,11 @@ export class StartLoadTestWidgetComponent {
         this.advanceForm.get('rampUp').value,
         this.advanceForm.get('followRedirect').value,
       );
+      sessionStorage.setItem('loadTestConfig', JSON.stringify(this.LTC));
```

Comment:
This might be acceptable for now, but in the future, this would likely be refactored into a service of some kind.